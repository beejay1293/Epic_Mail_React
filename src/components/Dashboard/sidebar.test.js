import React from 'react';
import { mount, shallow, render } from 'enzyme';
import Sidebar from './sidebar';
describe('Single button', () => {
  it('should render correctly the sidebar', () => {
    const Sbar = shallow(<Sidebar />);
  
    expect(Sbar).toMatchSnapshot();
  });
});
