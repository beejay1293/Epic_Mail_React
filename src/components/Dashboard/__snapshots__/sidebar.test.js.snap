// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Single button should render correctly the sidebar 1`] = `
<div
  className="nav"
>
  <Compose />
  <hr />
  <Connect(Btn) />
</div>
`;
